conf/docker/dataverse-glassfish/init-container/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM centos:7.2.1511
+MAINTAINER Dataverse (support@dataverse.org)
```

Comment:
Can we have a comment here about what "init-container" does? What it's purpose it?